cli/util/execute.go

```diff
@@ -0,0 +1,164 @@
+package util
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/mattn/go-isatty"
+
+	"github.com/apex/log"
+	"github.com/briandowns/spinner"
+)
+
+type emptyStruct struct{}
+
+// readyChannel is a channel used to signal completion of command execution.
+type readyChannel chan emptyStruct
```

Comment:
I would rename it to readyChan, sounds clear and more compactly and similar with chan type. But I don't insist, just idea. Feel free to ignore.